content/cpp/concepts/vectors/terms/erase/erase.md

```diff
@@ -0,0 +1,97 @@
+---
+Title: '.erase()'
+Description: 'Removes a single element or a range of elements from a vector.'
+Subjects:
+  - 'Computer Science'
+  - 'Game Development'
+Tags:
+  - 'Vectors'
+  - 'Arrays'
+  - 'Data Structures'
+CatalogContent:
+  - 'learn-c-plus-plus'
+  - 'paths/computer-science'
+---
+
+The **.erase()** function removes a single element or a range of elements from a specific position of the `vector`.
+When an element is remove with *.erase()* method, the size of the `vector` decrease. Therefore the elements after the ones deleted are moved to a new position.
+The position of the elements to remove are identified with `.begin()` and/or `.end()` functions.
```

Comment:
```suggestion
The **`.erase()`** function removes a single element or a range of elements from a specific position in the `vector`. When an element is removed with the `.erase()` function, the size of the `vector` decreases, and the elements after the deleted ones are shifted to fill the gap. The positions of the elements to remove are identified with `.begin()` and/or `.end()` functions.
```

Reply:
Modifications:
1. the first instance of the term should be in **bold** and `backticks`
2. Reframed the sentence for better clarity, it had some spelling mistakes
3. Every time we use a term, always put it in backticks: `erase()`, `.end()`, etc
4. We can either call it as a function or a method, so I changed the instance of method to function in this sentence.